kiliautoml/models/_pytorchvision_image_classification.py

```diff
@@ -194,27 +211,56 @@ def find_errors(
     ) -> Any:
         _ = clear_dataset_cache
 
-        original_image_datasets, labels, class_names = self.prepare_dataset(assets, api_key)
+        images = download_project_images(
+            api_key=api_key, assets=assets, output_folder=self.data_dir
+        )
+        labels = []
+        for asset in assets:
+            label = get_label(asset, label_merge_strategy)
+            if (label is None) or (self.job_name not in label["jsonResponse"]):
+                asset_id = asset["id"]
+                warnings.warn(f"${asset_id}: No annotation for job ${self.job_name}")
+                return 0.0
```

Comment:
why 0.0?

Reply:
It was because train need to return a float. But with find_errors, you are right it must return a list of assets so I changed it to []